pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -487,8 +487,12 @@ func (c *Cluster) configureMgrModules() error {
 
 		if module.Enabled {
 			if module.Name == balancerModuleName {
+				mode := module.Settings.BalancerMode
+				if mode == "" {
+					mode = balancerModuleMode
```

Comment:
How about renaming that default var?
```suggestion
					mode = defaultBalancerModuleMode
```